plugins/woocommerce/client/blocks/bin/webpack-config-interactivity.js

```diff
@@ -0,0 +1,72 @@
+/**
+ * External dependencies
+ */
+const path = require( 'path' );
+const DependencyExtractionWebpackPlugin = require( '@wordpress/dependency-extraction-webpack-plugin' );
+
+/**
+ * Internal dependencies
+ */
+const { sharedOptimizationConfig } = require( './webpack-shared-config' );
+
+// Config to build and incubate the interactivity API within WooCommerce.
+module.exports = {
+	entry: {
+		'@wordpress/interactivity': path.resolve(
+			__dirname,
+			'..',
+			'node_modules/@wordpress/interactivity/src/index.ts'
```

Comment:
We would still need to add the entry point that contains `preact/debug` (`debug.ts`), but we can do it later if its really needed.

Reply:
Yep, I'm thinking for now to just leave this. We can add later if needed as you say.